VagrantProvision.sh

```diff
@@ -27,65 +28,101 @@ if ! dpkg -l | grep --quiet wamerican-insane; then
     sudo dpkg-reconfigure dictionaries-common
 fi
 
-# Install deps for Cucumber tests
-sudo apt-get install ruby ruby-dev xvfb
-sudo gem install mime-types -v 2.6.2
-sudo gem install capybara -v 2.5.0
-sudo gem install cucumber capybara-webkit selenium-webdriver rspec capybara-screenshot
```

Comment:
There doesn't appear to be any harm in calling the gem install multiple times. Why replace 4 lines with 40? It may take a little longer, but provisioning doesn't happen very often. Also, the if statements below for the version specific installs may not work. (E.g. if mime-types v2.5 is installed, it won't be upgraded to v.2.6.2)